### PR TITLE
mkfs.erofs: add page

### PR DESCRIPTION
**Pull request by muxelplexer**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/linux/mkfs.erofs.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# mkfs.erofs
+
+> Create a erofs filesystem in an image.
+> More information: <https://erofs.docs.kernel.org/en/latest/>
```
Comment (spageektti) [inline pages/linux/mkfs.erofs.md:4]:
```suggestion
> More information: <https://erofs.docs.kernel.org/en/latest/>.
```

File: pages/linux/mkfs.erofs.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Create a erofs filesystem based on a root directory:
+
+`mkfs.erofs image.erofs root/`
+
+- Create a erofs image with a specific uuid:
+
+`mkfs.erofs -U 09307057-78f6-4dc3-b1b0-4a45e815a8f8 image.erofs root/`
```
Comment (spageektti) [inline pages/linux/mkfs.erofs.md:12]:
```suggestion
`mkfs.erofs -U {{uuid}} image.erofs root/`
```

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution. I left some minor suggestions.

**Comment by muxelplexer** [comment]
i have incorporated the suggestions

File: pages/linux/mkfs.erofs.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Create a erofs image with a specific uuid:
+
+`mkfs.erofs -U {{uuid}} image.erofs root/`
+
+- Create a compressed erofs image:
+
+`mkfs.erofs -zlz4hc image.erofs root/`
+
+- Create a erofs image where all files are owned by root:
```
Comment (kbdharun) [inline pages/linux/mkfs.erofs.md:18]:
```suggestion
> Create an EROFS filesystem in an image.
> More information: <https://erofs.docs.kernel.org/en/latest/>.

- Create an EROFS filesystem based on the root directory:

`mkfs.erofs image.erofs root/`

- Create an EROFS image with a specific UUID:

`mkfs.erofs -U {{uuid}} image.erofs root/`

- Create a compressed EROFS image:

`mkfs.erofs -zlz4hc image.erofs root/`

- Create an EROFS image where all files are owned by root:
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

File: pages/linux/mkfs.erofs.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+> Create a erofs filesystem in an image.
+> More information: <https://erofs.docs.kernel.org/en/latest/>.
+
+- Create a erofs filesystem based on a root directory:
+
+`mkfs.erofs image.erofs root/`
+
+- Create a erofs image with a specific uuid:
+
+`mkfs.erofs -U {{uuid}} image.erofs root/`
```
Comment (spageektti) [inline pages/linux/mkfs.erofs.md:12]:
```suggestion
`mkfs.erofs -U {{UUID}} image.erofs root/`
```

**Review by spageektti — APPROVED** [review]

**Pull request merged**